### ISSUE TITLE
XDY/XDV* recombinant with N:P279S and Orf1a:P971T, P1786S, S3244L (4, US, 4 states) and its recombinant with KP.3.1.1* with additional Orf1a:T3579I (4, Finland)

### ISSUE THREAD

**Issue by Walde1nsamkeit** [sars-cov-2-variants/lineage-proposals#2291]
## XDY/XDV* Recombinant
Query: C5621T, C9996T, T28297C
Breakpoint: 24872(25036?)\~28297
Mutations based on XDY:
**C3176A(Orf1a:P971T), C5621T(Orf1a:P1786S)**, C7681T, **C9996T(Orf1a:S3244L), A10699G, A17916G** / _T28297C, A28958Crev_, C29108T(N:P279S)
5' donor: XDY> C3176A, C9996T, A10699G, A17916G> C5621T
3' donor could be XDV*, XBB.1.9* or their related recombinants, but only XDV* is still circulating.
There are several XDV* or XBB.1.9* subbranch with C29108T, and XDV.1> T2572C, C7056T, C12781T, T17772C, A22995Crev, T25036C, C29108T (being placed under XDV.1> A22995Crev branch, #2114) is the largest one among the current circulating branches, mainly found in China but also in US-CA.
C7681T is private.

Earliest sequence: 11-08 Tennessee, United States EPI_ISL_19632555
Latest sequence: 02-05 Massachusetts, United States EPI_ISL_19739494
Regions detected: US 4(California, Massachusetts, Montana, Tennessee)

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_3c47e8_1e0a60.json?c=country&label=id:node_7126068
![Image](https://github.com/user-attachments/assets/e0cf80b8-710e-4cbb-92cd-f24f82344bf5)

Sequences:
EPI_ISL_19632555, EPI_ISL_19683605, EPI_ISL_19717071,
EPI_ISL_19739494

## KP.3.1.1*/(XDY/XDV*) Recombinant
Query: A13121T, G22111T, T28297C or T15837C, G19669A, G22131C by @FedeGueli 
Mutations based on JN.1:
_C7113T_, C11001T(Orf1a:T3579I), _C12616T, A13121T, G15372T_, T15837C, _G17334T / G19669A, del21653-21655, G22111T, G22131C, C22858T, T22928C, C23039G, G24872T, T28297C, A28958Crev_, C29108T(N:P279S)
Breakpoint: 17916\~19669

C11001T could be homoplastic or artefactual.
T15837C is private.

10-21~01-28 Finland 4

> @FedeGueli :
Quick snapshot of mutations
![image](https://github.com/user-attachments/assets/a646d0de-f515-4a1e-94e3-cd7b4025c595)
Basically it is XDY spike plus Orf9b/N from XDV.1 and Orf1ab from Kp.3.1.1

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_e4863_c23fe0.json?f_userOrOld=uploaded%20sample&label=id:node_11868675
![Image](https://github.com/user-attachments/assets/ff9f4503-7ff7-40b0-9a96-827111afa53f)

Sequences:
EPI_ISL_19612647, EPI_ISL_19612698, EPI_ISL_19718013,
EPI_ISL_19718018

**Comment by FedeGueli** [comment]
I have added query , tree and sanpshot of mutations.

**Comment by FedeGueli** [comment]
@Over-There-Is could be this the KP.3.1.1 donor? : https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice2_genome_test_20c0b_14dff0.json
<img width="870" alt="Screenshot 2024-12-17 alle 11 20 23" src="https://github.com/user-attachments/assets/63eabcf7-5e10-40d3-9203-4701551399cf" />

**Comment by Over-There-Is** [comment]
> @Over-There-Is could be this the KP.3.1.1 donor? : https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice2_genome_test_20c0b_14dff0.json <img alt="Screenshot 2024-12-17 alle 11 20 23" width="870" src="https://private-user-images.githubusercontent.com/87669813/396457126-63eabcf7-5e10-40d3-9203-4701551399cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NTM4ODgsIm5iZiI6MTczNDQ1MzU4OCwicGF0aCI6Ii84NzY2OTgxMy8zOTY0NTcxMjYtNjNlYWJjZjctNWUxMC00MGQzLTkyMDMtNDcwMTU1MTM5OWNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDE2Mzk0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkNDRiMzBlYTkzZTNkYTE1ODdlYmYwNGM2YzJjZGU1OWFmOGZiZGI4OWM3ZDZmZWQ4MjY4OGU3ODhmOWE4Y2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uuV0vIfGoXMfd6noHBrPxsMQPrTWPozL-kV8j8UkwGE">

I also found it but this recombinant doesn't have C5986T.

**Comment by Over-There-Is** [comment]
I don't think the tree could interpret its real evolution path so didn't attach it.

**Comment by FedeGueli** [comment]
> I don't think the tree could interpret its real evolution path so didn't attach it.

yeah as usual for recombinants.

**Comment by Over-There-Is** [comment]
Neither A13121T, C22858T, G22111T, -T28297C nor -A13121T, C22858T, G22111T, T28297C give meaningful result. And C12616T, A13121T, T28297C, -C22858T, -C28958A only gives 1 result too. It means there aren't any related KP.3.1.1/XDY, XDY/XDV or KP.3.1.1/XDV recombinants being detected.

**Comment by xz-keg** [comment]
no new seqs

**Comment by FedeGueli** [comment]
dead

**Comment by Over-There-Is** [comment]
4 now, 01-28

**Comment by JosetteSchoenma** [comment]
I noticed the 2 new ones while scanning for Delta with G210T. The dates of the 2 old and the 2 new ones are 3 months apart and one of the old plus one of the new ones are from women with the same age?
EPI_ISL_19612698 and EPI_ISL_19718013
Could this be the same woman with a chronic infection infecting someone else twice, 3 months apart?

**Comment by xz-keg** [comment]
no more seqs

**Comment by Over-There-Is** [comment]
Updated

**Comment by xz-keg** [comment]
no new seqs
